tools/project_manager/remote_file_manager.php

```diff
@@ -1075,33 +1078,31 @@ function show_message($type, $message)
 /**
  * Display a return link (to the 'showdir' view)
  */
-function show_return_link($relpath = null)
+function show_return_link(): void
```

Comment:
None of the callers passed the only argument so I removed it and simplified the code below.